osdconfig/proto/config.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+// this package defines the configuration parameters and interfaces
+// requires to run portwork services
```

Comment:
change should not be specific to portworx as such.

Reply:
thanks @sangleganesh , will do